commands/promote.js

```diff
@@ -0,0 +1,25 @@
+var fs = require("fs");
+
+module.exports = {
+  promoteCommand: function (message, roster, leaders) {
+    // if (leaders.length < 2) {
```

Comment:
Comments?

Reply:
Yea i don't know what y'all want to do with the promote function because it assumes 2 groups is the max and you cant promote some other member to run groups 3, 4, 5... What if theres a whale guild that can run more groups? then having a "there's already 2 groups" rejection kinda wont work for them. if users are manually promoting and demoting the exped leaders, there doesnt need to be a hard limit since they know the limit themselves. it might mess with group or balance logic though. I havent looked into those yet.

TLDR: just wanted confirmation from tom the bomb that this was the right decision to remove the max of 2 leader restriction message. nothing changed in the logic.

Reply:
In the groups rewrite I'm only touching !groups and balance, but I think the removal of this restriction makes sense. It should be an arbitrary amount of leaders and the app should just do the right thing

Reply:
Remove the comments for now and leave as is until Chris/me refactors for multiple groups imo

Reply:
I'm adding some null and more than 1 input checks if that's okay.